src/elements/common/annotator-context/types.ts

```diff
@@ -37,6 +44,14 @@ export type GetMatchPath = (location?: Location) => match<MatchParams> | null;
 export interface AnnotatorContext {
     emitActiveAnnotationChangeEvent?: (id: string) => void;
     emitAnnotationRemoveEvent?: (id: string, isStartEvent?: boolean) => void;
+    emitAnnotationReplyCreateEvent?: (
+        reply: Object,
+        requestId: string,
+        annotationId: string,
+        isStartEvent?: boolean,
+    ) => void;
+    emitAnnotationReplyDeleteEvent?: (id: string, annotationId: string, isStartEvent?: boolean) => void;
```

Comment:
Do we full capitalize id in camelcased names? Something like `annotationID`? Not sure if we try to push for this in BUIE

Reply:
I've checked and I haven't found in BUIE any case of the fully capitalized ID in variable names specifically for annotation or comment and just a few examples of such variables in other areas. Having in mind that it's currently broadly used with camel case, I don't think it makes sense to change all those instances at this point.